ledger-service/http-json/src/main/scala/com/digitalasset/http/domain.scala

```diff
@@ -174,7 +174,7 @@ object domain extends com.daml.fetchcontracts.domain.Aliases {
   final case class CreateUserRequest(
       userId: String,
       primaryParty: Option[String],
-      rights: List[UserRight],
+      rights: Option[List[UserRight]],
```

Comment:
What is the difference in semantics between a `None` and a `Some(Nil)`?

Reply:
There isn't any, however to not have to write json encoders/decoders by hand I decided to adjust the type to this.